### ISSUE TITLE
Abandoned Project

### ISSUE THREAD

**Issue by sparxooo** [IMS-MAXIMS/openMAXIMS#35]
It's a shame that with all the good intentions that IMS-Maxims had by releasing the source code, that the Open Source version hasn't been updated for so long.

Obviously any closed source project being released as open source will have it's teething problems, especially with any particular coding styles, or configuration programs used that don't get released, however it's been 2 years since the last update. Improvements, extra features, and most importantly security fixes can be found by the community, if only IMS-Maxims would keep the open source codebase up to date.

Commitment by the community can only be given when the project maintainers keep up their side of it too.

**Comment by pacharanero** [comment]
Yes this is disappointing. @ThiyagarajanJayaraman is seemingly the only committer on GitHub.

I understand that internally IMSMAXIMS don't actually use Git for version control, so I think the code on GitHub is a codebase which cannot be merged into the 'production' code. Which begs the question - what is the point of having a GitHub repo at all? #openwashing

**Comment by sparxooo** [comment]
Wow, didn't think i'd posted this almost 2 weeks ago. Sorry I didn't see your reply earlier. Lets hope that some movement is seen soon, maybe they are working behind the scenes towards the right direction. Lets also hope that they didn't just do this as a marketing ploy to get trusts to choose their software...

I'll keep an eye on the project, in the hope it might gain some traction...
